packages/sample-app-fabric/windows/SampleAppFabric/SampleAppFabric.cpp

```diff
@@ -47,20 +48,12 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
   // Enable per monitor DPI scaling
   SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
 
-  // Create a DispatcherQueue for this thread.  This is needed for Composition, Content, and
-  // Input APIs.
-  auto dispatcherQueueController{winrt::Microsoft::UI::Dispatching::DispatcherQueueController::CreateOnCurrentThread()};
-
-  // Create a Compositor for all Content on this thread.
-  auto compositor{winrt::Microsoft::UI::Composition::Compositor()};
-
   // Create a top-level window.
   auto window = winrt::Microsoft::UI::Windowing::AppWindow::Create();
```

Comment:
The app builder is already handling the window creation, right? By default most users won't be creating and setting app window themselves

Reply:
This is to showcase that they can create a window and pass it to the builder.

Reply:
I still think I'd rather the you set the title/size on `reactNativeWin32App.AppWindow()` after build but before start to showcase *that* location as the first preference for where they customize.